community/fulltext-index/src/main/java/org/neo4j/kernel/api/impl/fulltext/FulltextIndexAccessor.java

```diff
@@ -35,6 +35,7 @@
 import org.neo4j.values.storable.Values;
 
 import static org.neo4j.kernel.api.impl.fulltext.LuceneFulltextDocumentStructure.documentRepresentingProperties;
+import static org.neo4j.kernel.api.impl.fulltext.LuceneFulltextDocumentStructure.documentRepresentingPropertiesWithSort;
```

Comment:
This is just a style thing, and I see that the preexisting code was already doing this, but I really don't like importing single methods.

Since it follows the preexisting code style, I guess that's just the way it is, but just noting that I would rather see `import org.neo4j.kernel.api.impl.fulltext.LuceneFulltextDocumentStructure`, with `documentRepresentingPropertiesWithSort` then accessed as `LuceneFulltextDocumentStructure.documentRepresentingPropertiesWithSort()`.